src/kili/graphql/operations/project_user/queries.py

```diff
@@ -0,0 +1,61 @@
+"""
+GraphQL Queries of Project Users
+"""
+
+
+from typing import NamedTuple, Optional
+
+from kili.graphql import GraphQLQuery
+from kili.types import ProjectUser
+
+
+class ProjectUserWhere(NamedTuple):
```

Comment:
alternatively, the conversion to a GraphQL "where" dict can be done in this class. This would be more self-contained.

Reply:
Very good Idea! I didn't think about it